webapp/src/main/webapp/vue-apps/login-internal-onboarding/components/InternalOnboardingExpired.vue

```diff
@@ -19,8 +19,8 @@
 
 -->
 <template>
-  <v-card flat>
-    <v-card-title class="primary--text text-font-size px-0">
+  <v-card flat class="transparent">
```

Comment:
Attrs with attribute value goes before boolean attrs without value (Just a Hint)
+ you can use `color` on `v-card` instead of `class`

```suggestion
  <v-card color="transparent" flat>
```

Reply:
done